dencoding/yaml_decoder_test.go

```diff
@@ -98,12 +98,16 @@ key2: value6
 	})
 
 	t.Run("YamlAliases", func(t *testing.T) {
-		b := []byte(`foo: &foo
+		b := []byte(`foo: &foofoo
```

Comment:
The anchor and the key do not need to have the same value. This makes sure we don't actually use the anchor value for anything.